src/client/src/ui/footer/model/footerModel.js

```diff
@@ -31,13 +34,29 @@ export default class FooterModel extends ModelBase {
     this._subscribeToConnectionStatus();
   }
 
+  @observeEvent('toggleServiceStatus')
+  _onToggleServiceStatus() {
+    _log.debug(`toggling service status`);
+    this.shouldShowServiceStatus = !this.shouldShowServiceStatus;
+  }
+
+
   _subscribeToConnectionStatus() {
     this.addDisposable(
       this._compositeStatusService.serviceStatusStream.subscribeWithRouter(
         this.router,
         this.modelId,
         (serviceStatusLookup:ServiceStatusLookup) => {
+          const services = Object.keys(serviceStatusLookup.services)
+                                 .map(key => serviceStatusLookup.services[key]);
           this.serviceLookup = serviceStatusLookup;
+          if (_.every(services, service => service.isConnected)) {
```

Comment:
seems like this can probably be done in 1 loop rather than `.every` and `.some` - though given how few services they are, it does not matter. the api supports property shortcuts so you could say `_.every(services, 'isConnected')`